sky/utils/schemas.py

```diff
@@ -155,3 +155,43 @@ def get_task_schema():
             },
         }
     }
+
+
+def get_cluster_schema():
+    return {
+        '$schema': 'https://json-schema.org/draft/2020-12/schema',
+        'type': 'object',
+        'required': [],
```

Comment:
Should we use `'required': ['cluster', 'auth']`, as they are required?

Reply:
Done, thanks!